modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -39,8 +41,10 @@ const ClassicEditor = ({
 	title,
 }) => {
 	const editorRef = useRef();
+	const labeledToolbarsWidth = useRef();
```

Comment:
Maybe initial value should be set to 0 here, so `labeledToolbarsWidth.current` will always contain a number, even before `getToolbarsWidth()` is called.